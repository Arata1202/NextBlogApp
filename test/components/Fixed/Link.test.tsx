import { render } from '@testing-library/react';
import LinkPage from '@/components/Fixed/Link';

const mockSidebarArticles = {
  contents: [
    { id: '1', title: '記事1', description: '概要1', url: '/articles/1' },
    { id: '2', title: '記事2', description: '概要2', url: '/articles/2' },
    { id: '3', title: '記事3', description: '概要3', url: '/articles/3' },
  ],
};

describe('LinkPage', () => {
  test('スナップショット（LinkPage）', () => {
    const { asFragment } = render(<LinkPage sidebarArticles={mockSidebarArticles} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
