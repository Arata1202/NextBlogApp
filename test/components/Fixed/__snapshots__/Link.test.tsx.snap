// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinkPage スナップショット（LinkPage） 1`] = `
<DocumentFragment>
  <div
    class="max-w-[85rem] sm:px-6 lg:px-8 mx-auto pb-2"
  >
    <div
      class="grid lg:grid-cols-3 gap-y-8 lg:gap-y-0 lg:gap-x-6"
    >
      <div
        class="lg:col-span-2"
      >
        <div
          class="FirstAd"
        />
        <div
          class=""
        >
          <div
            class="space-y-5 lg:space-y-8"
          >
            <div
              class="includeBanner flex justify-end gap-x-5"
            />
          </div>
          <div
            class="content mt-10 mb-5"
          >
            <h2
              class="bg-gray-300 text-gray-700"
              id="link"
            >
              リンクについて
            </h2>
            <p>
              リアル大学生（以下、「本サービス」と言います。）は完全リンクフリーです。リンクを行う場合の本サービスへの許可や連絡は不要です。
              <br />
              <br />
              ただし、インラインフレームの使用や画像の直リンクはご遠慮ください。
            </p>
          </div>
        </div>
        <div
          class="FirstAd"
        />
      </div>
      <div
        class="mobile"
      />
    </div>
  </div>
</DocumentFragment>
`;
